rpi_backlight/__init__.py

```diff
@@ -70,7 +74,6 @@ def __init__(
         self._backlight_sysfs_path = Path(backlight_sysfs_path)
         self._board_type = board_type
         self._fade_duration = 0.0  # in seconds
-
```

Comment:
Pointless whitespace change

Reply:
Sorry about this one! My mistake.

Reply:
Fixed in https://github.com/p1r473/rpi-backlight/commit/797b159eeb11df9e35013e81418b69f6d4e08af2

Reply:
No, now you added whitespace. Please take a look at the diff before resolving comments

Reply:
Sorry about that, I think I got it now
Fixed in https://github.com/p1r473/rpi-backlight/commit/cb15998e38f82722352a08c2573188ca9653d839
Thank you for your patience